sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
actually, I changed package cmd->main in the other files  - kept it as main in sc-cli.go